playground/playground.css

```diff
@@ -60,14 +56,15 @@
   margin-bottom: 0;
 }
 
-#page-playground .editor-option .btn {
-  padding: 1px 5px 0px 5px;
-}
-
 #page-playground .read-only {
   opacity: .6;
 }
 
+#page-playground .btn {
+  height: 30px;
+  padding: 0 0.7em;
+}
```

Comment:
Ideally, there wouldn't be anything custom required here at all. If the buttons still look wrong, we must not be doing something quite right with bootstrap. There default theme looks just fine (and we're trying to use that, AFAIK).